src/excmds.ts

```diff
@@ -2511,22 +2521,24 @@ export async function fullscreen() {
 */
 //#background
 export async function tabclose(...indexes: string[]) {
-    let done
     async function maybeWinTabToTabId(id: string) {
```

Comment:
Since you're making these changes, the function name should be renamed to remove `Id` at the end